spec/plans/execute.fmf

```diff
@@ -217,33 +217,6 @@ description: |
       - implemented-by: /tmt/steps/execute/internal.py
       - verified-by: /tests/execute/framework
 
-/where:
-    summary: Execute tests on selected guests
-    description: |
-        In the :ref:`/spec/plans/provision/multihost` scenarios it
-        is often necessary to execute test code on selected guests
-        only or execute different test code on individual guests.
-        The ``where`` key allows to select guests where the tests
-        should be executed by providing their ``name`` or the
-        ``role`` they play in the scenario. Use a list to specify
-        multiple names or roles. By default, when the ``where`` key
-        is not defined, tests are executed on all provisioned
-        guests.
-    example: |
-        # Execute discovered tests on the client (already supported)
-        execute:
-          - how: tmt
-            where: client
-
-        # Run different script for each role (not supported yet)
-        execute:
-          - name: run-the-client-code
-            script: client.py
-            where: client
-          - name: run-the-server-code
-            script: server.py
-            where: server
-
```

Comment:
So we are removing this from the spec however, the basic functionality is already there, see the following test:

https://github.com/teemtee/tmt/blob/bdfcd71bf44bcc8968783eb458f93369cf7e3519/tests/multihost/web/data/plan.fmf#L33-L35

Shall we silently drop the functionality or try to be backward-compatible for those who already started experimenting? @lukaszachy, @thrix, @happz, @FrNecas, @adiosnb, any thoughts on this?

Reply:
Discussed shortly on the last hacking session and agreed to keep the current functionality until the `where` keyword is supported under the `discover` step and only then obsolete it.